aces/Loris
# [mindlogger] Implementation ML integration POC1

## Brief summary of changes

- [x] Implemented module for receiving ML applet schemas, sving in the Loris db and creating instruments.
- [x] Updated base instrument class to handle ML applet instruments.
- [x] Added base ML instrument class to handle to process ML applet schemas


#### Testing instructions 

1. Login to the Loris API with user that have right "mindlogger_schema_create"
2. Make authorized request to Post: /mindlogger/v1/schema/ with valid applet schema.
3. Go to '$LORISURL/instrument_manager/'. You should see new ML instrument there
4. Got to '$LORISURL/dictionary/'. You should see new ML instruments fields in the fields list
